IRCT-CL/src/main/java/edu/harvard/hms/dbmi/bd2k/irct/cl/util/Utilities.java

```diff
@@ -224,7 +232,13 @@ public static String extractUserFromTokenIntrospection(HttpServletRequest req, S
 
 		ObjectMapper json = IRCTApplication.objectMapper;
 		CloseableHttpClient client = IRCTApplication.CLOSEABLE_HTTP_CLIENT;
-		HttpPost post = new HttpPost(token_introspection_url);
+
+		String[] token_intro_urlArray = token_introspection_url.substring(8).split("/");
+		HttpHost target = new HttpHost(token_intro_urlArray[0], 443, "https");
+
+		HttpPost post = new HttpPost(token_introspection_url.substring(8+token_intro_urlArray[0].length()));
```

Comment:
Notthing specific, just to make code look clear: there should be space in left and right of `+`.